### ISSUE TITLE
Place bets are automatically turned off when point is Off

### ISSUE THREAD

**Issue by amortization** [skent259/crapssim#10]
I believe that at most craps tables they are automatically turned off, but you do have the option to tell them to leave them working if you desire. I think that Place bets should give the option to take them down or be left up when the point is working, otherwise, have them automatically left up and taken down when the point is off based on the players strategy.

**Comment by skent259** [comment]
I like the idea of allowing an option for having the place bets left-up and "on" for the come-out rolls, though I still think the default should be that they are left-up but "off".  This option should be player/strategy specific. 

Just to clarify my terminology:
- on: bet is on table and working (i.e. at risk)
- off: bet is on table but not working, bet doesn't update
- left-up: bet stays on table, but could be either on/off
- taken-down: bet amount returns to player, and if removed from the table

I think it would be counterintuitive if place bets were taken-down after the point is hit, but maybe I misinterpreted your suggestion?

**Comment by amortization** [comment]
Yeah, I should have been more clear in my original comment, I think that Place bets (as well as other bets like Come Odds since I know a lot of places turn those off when the point is off too) should have the option of being turned On/Off depending on the Point. I think that the leaving up/taking down should still be the current logic.

**Comment by skent259** [comment]
This behavior is now available in the StrategyMode and strategy.tool as of #55.

The default `BetPlace` will treat point bets like most tables, they are not working (off) when the point is off. `StrategyMode.BET_IF_POINT_ON`. Functionally, the strategy will remove the place bet when point is off, which differs slightly from how a real table works, but it means we don't need separate functionality within the place bet.

To have place bets working while the point is off, use `StrategyMode.ADD_IF_POINT_ON` or `StrategyMode.ADD_IF_NON_EXISTENT`

In the future, may consider a "keep working" option and changing default place bet update function, but closing for now.
